wsd/HostUtil.cpp

```diff
@@ -143,19 +143,26 @@ void HostUtil::parseAliases(Poco::Util::LayeredConfiguration& conf)
                 break;
             }
 
+            auto aliasString = conf.getString(aliasPath, "");
+            if (aliasString.empty())
+            {
+                continue;
+            }
+
             try
             {
-                const Poco::URI aliasUri(conf.getString(aliasPath, ""));
-                if (aliasUri.empty())
-                {
-                    continue;
-                }
+                const Poco::URI aliasUri(aliasString);
 
-                for (const std::string& x : Util::splitStringToVector(aliasUri.getHost(), '|'))
+                for (const std::string& aliasPattern : Util::splitStringToVector(aliasUri.getHost(), '|'))
                 {
-                    const Poco::URI aUri(aliasUri.getScheme() + "://" + x + ':' +
+                    if (!RegexUtil::isRegexValid(aliasPattern)) {
+                        LOG_WRN ("parseAliases: Invalid regex alias [" << aliasPattern << "] for host" << uri);
```

Comment:
Is it possible to issue this warning multiple times, or is this utility called only once? Ideally we warn once for each invalid pattern at start-up and ignore them from then on.

Reply:
Ideally we want to add a warning in the audit window too.